src/Learn/Component.svelte

```diff
@@ -62,7 +66,7 @@
 </style>
 
 <div
-    class="max-h-page flex flex-col items-center text-primary mx-auto object-contain"
+    class="h-full flex flex-col items-center text-primary mx-auto object-contain"
```

Comment:
This probably leads to the following (see the image)
This is not easy but let's find a way for it to take the whole space but not got outside its parent's borders
![image](https://user-images.githubusercontent.com/15484066/151498332-36c7a3e7-8c99-4711-b652-1c6bcd274e9a.png)